src/js/index.js

```diff
@@ -0,0 +1,72 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const home = document.getElementById('sliderHome')
+  const services = document.getElementById('services')
+  const portfolio = document.getElementById('portfolio')
+  const about = document.getElementById('aboutPortfolio')
+  const contact = document.getElementById('contact')
+  const navbarPortfolio = document.getElementById('navbarPortfolio')
+  const navbarHome = document.getElementById('navbarHome')
+  const navbarservices = document.getElementById('navbarservices')
+  const navbarAbout = document.getElementById('navbarAbout')
+  const navbarContact = document.getElementById('navbarContact')
+  const headerHeight = document.querySelector('.wrapper__header').getBoundingClientRect().height
+  const scrollOptions = {
+    behavior: 'smooth',
+    block: 'start',
+  }
+
+  document.addEventListener('scroll', () => {
+    let scrolled = window.pageYOffset || document.documentElement.scrollTop
+    const arry = [
+      {
+        buttonId: 'navbarHome',
+        sectionId: 'sliderHome',
+      },
+      {
+        buttonId: 'navbarservices',
+        sectionId: 'services',
+      },
+      {
+        buttonId: 'navbarPortfolio',
+        sectionId: 'portfolio',
+      },
+      {
+        buttonId: 'navbarAbout',
+        sectionId: 'aboutPortfolio',
+      },
+      {
+        buttonId: 'navbarContact',
+        sectionId: 'contact',
+      },
+    ]
+    const get = (id) => document.getElementById(id)
```

Comment:
This function is also common and you don't have to create it each time, when user scrolls. Replace it outside.